app/models/user.rb

```diff
@@ -12,8 +18,9 @@ class User < ApplicationRecord
   before_create :set_access_token
 
   validates :full_name, presence: true
-  validates :phone_number, presence: true
+  validates :phone_number, presence: true, uniqueness: true
```

Comment:
Phone numbers for users need to be unique so that the user can be looked up with just phone number